pull-request-checklist.md

```diff
@@ -0,0 +1,10 @@
+## Pull Request Checklist
+
+This file outlines a list of common things that should be addressed before opening a PR. It's built from previous issues we've seen in a lot of pull requests. If you notice something that's being noted in a lot of PR's, it should probably be added here to help save people time in the future.
+
+- [ ] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in `src/sample_package` for examples.
+- [ ] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `src/sample_package`
```

Comment:
Could also ask to refer to [Coding Conventions](https://github.com/UBC-Snowbots/Snowflake#coding-conventions) for how to write these comments

Reply:
Hmm, not a bad idea, but I'm inclined to move towards putting all of this sort of documentation in `sample_package`, as that's what's usually coped when making new packages, and it would be nice to only have to update documentation in one place.